inst/examples/12_counter/tests/server/test-mymodule.R

```diff
@@ -0,0 +1,18 @@
+# Use testthat just for expectations
```

Comment:
Is this file required to be in the `./tests/server` folder?

Doesn't it work out fo the box with `./tests/testthat`?